app/assets/stylesheets/partials/_header.css.scss

```diff
@@ -0,0 +1,20 @@
+header.navbar {
+  background-color: #232120;
+  border-radius: 0;
+
+  ul.navbar-nav li a, .nav-header a {
+    color: #f8f8f8;
+  }
+
+  ul.navbar-nav li a:hover {
+    background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.